numpydoc/docscrape.py

```diff
@@ -236,10 +236,30 @@ def _parse_param_list(self, content):
 
         return params
 
+    _role = r":(?P<role>\w+):"
+    _funcbacktick = r"`(?P<name>(?:~\w+\.)?[a-zA-Z0-9_.-]+)`"
+    _funcplain = r"(?P<name2>[a-zA-Z0-9_.-]+)"
+    _funcname = r"(" + _role + _funcbacktick + r"|" + _funcplain + r")"
+    _funcnamenext = _funcname.replace('role', 'rolenext').replace('name', 'namenext')
+    _description = r"(?P<description>\s*:(\s+(?P<desc>\S+.*))?)?\s*$"
```

Comment:
(I think we'd like to make the space before the colon obligatory, for consistency with param lists, but I don't think the current code requires it)

Reply:
I believe that the original `_name_rgx` did not require it.